AST-3438 - Fix: Locally downloaded google fonts not working properly in… (#5625)
brainstormforce/astra: changelog.txt

v4.5.3 (Unreleased)
- Fix: Logo color not working in FireFox browser.
- Fix: Site title, tagline font-sizes are not part of export-import header settings.

> - Fix: Locally downloaded google fonts not working properly in multisite WordPress setup.

Reviewer: ```suggestion
- Fix: Self-hosted Google fonts not working properly on the multisite setup.
```
